test/throws.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var lost    = require('../lost');
+var postcss = require('postcss');
+var CssSyntaxError = require('postcss').CssSyntaxError;
+
+module.exports = function throws( input, message, opts ) {
+  var processor = postcss([lost(opts)]);
+  expect(function(){
+    return processor.process(input).css;
+  }).to.throw(CssSyntaxError, message);
+};
```

Comment:
@codebysubtract Are you using the `.editorconfig` file? It should add new lines to the end of this file. This should absolutely be automated/linted if we're gonna "enforce" it–but it hopefully would help with having whitespace diffs.

I think this would be my only "nit-pick" in this PR.